### DISCUSSION TITLE
Comfy UI doesnot show load, save, options under queue node. HELP!!!!

### DISCUSSION THREAD

**Discussion by Zebronicsqwert1** [Comfy-Org/ComfyUI#6250] [General]
![Screenshot 2024-12-28 194344](https://github.com/user-attachments/assets/6d4fc75e-aafe-42d6-a0c0-51979ec3fe03)
Dear GitHub community, 
As you can see that the comfy UI  does not show the load, save and other options in under queue node.  Kindly help how to resolve it.

**Comment by LukeG89** [comment]
It's just new frontend!

The load and save options are moved on topbar:

![image](https://github.com/user-attachments/assets/2e7c81b9-0aa6-40c3-b954-9a23c221a01b)


If you really want the old legacy menu back, you can disable the new menu in Settings:

![image](https://github.com/user-attachments/assets/4ef8c792-fba5-4b69-beb5-ca3dc8d9904a)

**Comment by LukeG89** [comment]
Anyways, you can also dock the Queue menu on topbar if you like

![image](https://github.com/user-attachments/assets/dda5cb17-6c40-4aaf-a9f4-10eea5f2c096)

**Comment by BDEERAJ** [comment]
Can you suggest any free resources to learn frontend development? There are very few good resources on YouTube, and I need something better.

**Reply by ltdrdata** [reply]
An official development guide has not been released yet. However, you can find help here.
https://github.com/Comfy-Org/ComfyUI_frontend
